include/date/date.h

```diff
@@ -6838,7 +6838,7 @@ from_stream(std::basic_istream<CharT, Traits>& is, const CharT* fmt,
                                 auto wc = Traits::to_char_type(i);
                                 auto c = static_cast<char>(wc);
                                 // is c a valid time zone name or abbreviation character?
-                                if (!(CharT{1} < wc && wc < CharT{127}) || !(isalnum(c) ||
+                                if (!(CharT{1} < wc && wc < CharT{127}) || !(isalnum(i) ||
```

Comment:
I think this one is ok because of the preceeding `!(CharT{1} < wc && wc < CharT{127})`